### PR TITLE
New icon for PowerShell Core

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Fix https://github.com/PowerShell/PowerShell/issues/4676

Default color using familiar console icon, but with avatar (Posh-chan) colors

icon

### CODE REVIEW

**Comment by SteveL-MSFT** [comment]
Vote here https://github.com/PowerShell/PowerShell/issues/4676#issuecomment-332377214

**Comment by SteveL-MSFT** [comment]
Latest change is to add some customer bitmaps to the installer:

installer
page2

**Comment by jpsnover** [comment]
Looks great!

**Comment by iSazonov** [comment]
Can we make this woman's face beautiful? Only I think it skewed?

**Comment by SteveL-MSFT** [comment]
@iSazonov are you referring to the "mouth"?  that's the cursor of the prompt, sort of like `>_`

**Comment by iSazonov** [comment]
Yes,  "mouth", "nose" - I'd preffer see a nice face. Can we make from >_ a purse? :-)

**Comment by SteveL-MSFT** [comment]
@iSazonov I think it's too late to make changes to Posh-chan.  It came from a pro designer and it's what we've been using in some marketing.  I'm just reusing one of the original assets.  Think of it more as "war paint" on her face ;P

**Comment by SteveL-MSFT** [comment]
Last change is to fix the `Launch PowerShell` checkbox as the default control has a grey background which overlaps with the black icon.  Now it looks like

launch

File: assets/Product.wxs [hunk 1]
```diff
@@ -1,12 +1,10 @@
-﻿<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-
+﻿<Wix
+  xmlns="http://schemas.microsoft.com/wix/2006/wi">
```
Comment (adityapatwardhan) [inline assets/Product.wxs]: Unnecessary change?

**Review by adityapatwardhan — APPROVED** [review]
LGTM with minor comment.

File: assets/Product.wxs (hunk 1 above)
Comment (SteveL-MSFT) [reply assets/Product.wxs]: Might have accidentally hit Enter on that line.  Will fix.

**Comment by DimaRazb** [comment]
She seems very aggressive, I don't like her very much. Designer could use a purse or raincoat I believe.

**Comment by joeyaiello** [comment]
@SteveL-MSFT It's unclear to me which one is the default in this PR. Does it need to be updated to the black icon by default still? 

Otherwise, LGTM. Love the new MSI look-and-feel.

**Comment by SteveL-MSFT** [comment]
@joeyaiello default is:

blackicon

**Comment by SteveL-MSFT** [comment]
@PowerShell/powershell-committee reviewed this and fine with the changes

**Comment by adityapatwardhan** [comment]
@joeyaiello Can you update your review?

**Review by joeyaiello — APPROVED** [review]
Yup, I'm good
